docs/track.rst

```diff
@@ -19,7 +19,7 @@ A track is specified in a JSON file.
 Ad-hoc use
 ..........
 
-For ad-hoc use you can store a track definition anywhere on the file system and reference it with ``--track-path``, e.g::
+For ad-hoc use you can store a track definition anywhere on the file system and reference it with ``--track-path``, e.g.:
```

Comment:
this double-colon should be preserved https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks